ovos_bus_client/client/client.py

```diff
@@ -420,12 +420,12 @@ def on_message(self, *args):
         self.emitter.emit(parsed_message.msg_type, parsed_message)
 
 
+@deprecated("Bus monitoring moved to `neon-mana-utils`", "0.1.0")
```

Comment:
should we say no replacement? or maybe add a script entrypoint for this in this repo, just not an importable function

Reply:
I guess there isn't a direct replacement, so it makes sense to just log that